### PR TITLE
feat: add S390x support in affinity rule

### PR DESCRIPTION
**Pull request by modassarrana89**
Added s390x in node affinity rule as already work is going on to support s390x for tensorflow in runtime-adaptor

### CODE REVIEW

**Comment by modassarrana89** [comment]
@njhill @ckadner @rafvasq Can you guys please review & do the needful

File: controllers/modelmesh/runtime.go [hunk 1]
```diff
@@ -342,16 +342,15 @@ func (m *Deployment) addPassThroughPodFieldsToDeployment(deployment *appsv1.Depl
 	rts := m.SRSpec
 	// these fields map directly to pod spec fields
 	// supported architectures are "amd64" and "arm64", "ppc64le"
-	// and "s390x" are not supported by tensorflow
-	// (https://github.com/kserve/modelmesh-runtime-adapter/pull/38#discussion_r1156749259)
+	// and "s390x"
```
Comment (rafvasq) [inline controllers/modelmesh/runtime.go:345]: This comment needs some revision. It seems like the original comment says that both `pp64le` and `s390x` are not supported. The new comment is a bit confusing and looks like they are now both supported by this change. Can you please confirm and revise?

**Review by rafvasq — CHANGES_REQUESTED** [review]
Thanks @modassarrana89, based on the checks it seems like you also need to [lint your changes](https://github.com/kserve/modelmesh-serving/blob/main/docs/developer.md#formatting-and-linting-code) when you get a chance

File: controllers/modelmesh/runtime.go (hunk 1 above)
Comment (modassarrana89) [reply controllers/modelmesh/runtime.go:345]: Tensorflow support for s390x and pp64cle is still required & i am currently working on it. Here i have added s390x as part of node affinity. If it is still confusing , do let me know i will update the comment

Comment (rafvasq) [reply controllers/modelmesh/runtime.go:345]: Thanks for the info, but the comment still needs to be corrected either way. The change you made results in the comment:
```
supported architectures are "amd64" and "arm64", "ppc64le" and "s390x"
```

If all four are supported then it should be:
```
supported architectures are "amd64", "arm64", "ppc64le" and "s390x"
```

But it sounds like they're not, so maybe it's supposed to be something like the following with updated information on the unsupported arch:
```
supported architectures are "amd64", "arm64", and "s390x". "ppc64le" is not yet supported by tensorflow (https://github.com/kserve/modelmesh-runtime-adapter/pull/38#discussion_r1156749259).
```

Comment (modassarrana89) [reply controllers/modelmesh/runtime.go]: Updated comments in latest fix

**Review by rafvasq — CHANGES_REQUESTED** [review]
Sorry for the delay in returning to this @modassarrana89. It looks like you'll have to lint as before.

**Comment by ckadner** [comment]
@modassarrana89 -- I updated your branch and fixed the linting issue

**Review by ckadner — APPROVED** [review]
Nice! Thanks @modassarrana89

**Review by rafvasq — APPROVED** [review]

**Pull request merged**